src/net/FetchHttpClient.ts

```diff
@@ -115,7 +115,7 @@ export class FetchHttpClient extends HttpClient implements HttpClientInterface {
     path: string,
     method: string,
     headers: RequestHeaders,
-    requestData: RequestData,
+    requestData: string,
```

Comment:
Anyone know why we went from `RequestData` which is `Record<string, any>` to `string` but none of the places where this is consumed had to be changed?